packages/webdriverio/src/commands/element/getHTML.js

```diff
@@ -22,7 +22,7 @@
  * </example>
  *
  * @alias element.getHTML
- * @param {Boolean=} includeSelectorTag if true it includes the selector element tag (default: true)
+ * @param {Boolean} includeSelectorTag if true it includes the selector element tag (default: true)
```

Comment:
I am not sure if this is the right solution to this problem. The `=` at the end of the type marks the parameter as optional we I think we should keep.

Reply:
Ah, OK, then let's explore modifying the template as you suggested above. I can change this back. :)